go/api/base_client.go

```diff
@@ -503,27 +506,27 @@ func (client *baseClient) Set(key string, value string) (string, error) {
 //
 // Return value:
 //
-//	If the value is successfully set, return api.Result[string] containing "OK".
+//	If the value is successfully set, return modelsResult[string] containing "OK".
 //	If value isn't set because of ConditionalSet.OnlyIfExists or ConditionalSet.OnlyIfDoesNotExist
-//	or ConditionalSet.OnlyIfEquals conditions, return api.CreateNilStringResult().
+//	or ConditionalSet.OnlyIfEquals conditions, return modelsCreateNilStringResult().
 //	If SetOptions.returnOldValue is set, return the old value as a String.
 //
 // [valkey.io]: https://valkey.io/commands/set/
-func (client *baseClient) SetWithOptions(key string, value string, options options.SetOptions) (Result[string], error) {
+func (client *baseClient) SetWithOptions(key string, value string, options options.SetOptions) (models.Result[string], error) {
 	optionArgs, err := options.ToArgs()
 	if err != nil {
-		return CreateNilStringResult(), err
+		return models.CreateNilStringResult(), err
 	}
 
 	result, err := client.executeCommand(C.Set, append([]string{key, value}, optionArgs...))
 	if err != nil {
-		return CreateNilStringResult(), err
+		return models.CreateNilStringResult(), err
 	}
 
 	return handleOkOrStringOrNilResponse(result)
 }
 
-// Get string value associated with the given key, or api.CreateNilStringResult() is returned if no such value
+// Get string value associated with the given key, or modelsCreateNilStringResult() is returned if no such value
```

Comment:
Same here and below (and maybe in other files?)
I propose to do search-and-replace with a regex like `//(.*)modelsCreate(.*) -> //$1models.Create$2`